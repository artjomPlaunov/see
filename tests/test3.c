int func () {
  int a = 5, b=6; 
  int var = 7, b=8;
}

int main () {
  int variable = 132312321, fefew = 4343232;

}
