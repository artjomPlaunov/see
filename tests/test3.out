(Ast.Prog
   [(Ast.Fun (Ast.INT_T, (Ast.ParamLst []), (Ast.Ident "func"),
       (Ast.Block
          [(Ast.Stm_declLst (Ast.INT_T,
              [(Ast.Decl ((Ast.Ident "a"), 5));
                (Ast.Decl ((Ast.Ident "b"), 6))]
              ));
            (Ast.Stm_declLst (Ast.INT_T,
               [(Ast.Decl ((Ast.Ident "var"), 7));
                 (Ast.Decl ((Ast.Ident "b"), 8))]
               ))
            ])
       ));
     (Ast.Fun (Ast.INT_T, (Ast.ParamLst []), (Ast.Ident "main"),
        (Ast.Block
           [(Ast.Stm_declLst (Ast.INT_T,
               [(Ast.Decl ((Ast.Ident "variable"), 132312321));
                 (Ast.Decl ((Ast.Ident "fefew"), 4343232))]
               ))
             ])
        ))
     ])